org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/thingTranslator.js

```diff
@@ -93,7 +94,7 @@ class Translator {
         var putPayloadString = JSON.stringify(putPayload);
         return this._makeRequest(endpointUri, this._updatePath + '/' + deviceId, 'PUT', putPayloadString)
             .then((result) => {
-                if (result === "succeed") {
```

Comment:
This is align with the status return from POST methods returned from SmartApp.

---
In reply to: [109978582](https://github.com/openT2T/translators/pull/225#discussion_r109978582) [](ancestors = 109978582)